api/disabled_parameters_test.go

```diff
@@ -0,0 +1,447 @@
+package api
+
+import (
+	"errors"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/require"
+)
+
+type testingErrorReporter struct {
+	ErrorFCalledCount int
+}
```

Comment:
I do like the error reporter defined here because I get to count how many times it was called as opposed to the log message (which might change in the future).

Reply:
The null logger "hook" has all log entries, so you can count the size of that array

Reply:
Done